tests/e2e/scale/test_scale_3_OCS_worker_nodes_and_1500_PVCs.py

```diff
@@ -111,6 +111,70 @@ def test_pv_scale_out_create_pvcs_and_respin_ceph_pods(
         )
 
 
+@scale
+@ignore_leftovers
+@skipif_external_mode
+@ipi_deployment_required
+@pytest.mark.parametrize(
+    argnames="resource_to_delete",
+    argvalues=[
+        pytest.param(*["noobaa_operator"], marks=[pytest.mark.polarion_id("OCS-4880")]),
+        pytest.param(*["ocs_operator"], marks=[pytest.mark.polarion_id("OCS-4880")]),
+        pytest.param(*["odf_operator"], marks=[pytest.mark.polarion_id("OCS-4880")]),
+        pytest.param(*["operator"], marks=[pytest.mark.polarion_id("OCS-4880")]),
+    ],
+)
+class TestScaleRespinOperatorPods(E2ETest):
+    """
+    Respin Operator pods in cluster with 1500+ PVC & 76 Pods
+    operator ==> OPERATOR_LABEL = "app=rook-ceph-operator"
+    ocs-operator ==> OCS_OPERATOR_LABEL = "name=ocs-operator"
+    odf-operator ==> ODF_OPERATOR_CONTROL_MANAGER_LABEL = "control-plane=controller-manager"
+    noobaa-operator ==> NOOBAA_OPERATOR_POD_LABEL = "noobaa-operator=deployment"
+    """
+
+    def test_respin_operator_pods(
+        self,
+        fioscale,
+        resource_to_delete,
+    ):
+        """
+        Test case to respin operator pods in cluster with scaled PVC+POD's counts
```

Comment:
are the pods created in the same namespace or different namespaces?

Reply:
Different namespace

Reply:
Discussed with Ram offline, it seems the BZ is not dependent on the namespace. so the all the resources can be created in the same namespace